### DISCUSSION TITLE
Which hardware for a Tasmota PoE device?

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#19011] [Q&A] [answered]
Hello
want to implement an ESP32 device running Tasmota to measure water level using a JSN SR-40T sensor. But there is no WIFI... therefore maybe a device with ETH and POE.
Which device is recommended for this purpose?
.) [LILYGO T-Internet-POE ESP32-WROOM LAN8720A](https://www.amazon.de/LILYGO-T-Internet-POE-ESP32-WROOM-Erweiterungsplatine-programmierbare/dp/B0BHYT98LN?ref_=ast_sto_dp&th=1)
.) [Olimex ESP32-POE-ISO](https://www.amazon.de/Olimex-ESP32-POE-ISO/dp/B08NWLHH8V/ref=sr_1_1?keywords=Olimex&qid=1688307086&s=computers&search-type=ss&sr=1-1)
.) ...
Thanks!

**Comment by barbudor** [comment, marked as answer]
Both supports Tasmota
The Olimex version you pointed is the isolated one. Which is safer but you can save some euros on the standard version if you don't need that additional security level.

If you don't need the SD card, I would suggest the Olimex one as it is smaller form factor and manufactured in Europe.
I'm not using any of those, so just an opinion.

**Reply by sfromis** [reply]
Unless you also buy the optional adapter, the LilyGo one is a bit finicky to flash, the 6 holes at 2.0 mm pitch near the ESP32 module are the needed pins, but unlabeled. Seen from the module, they are 3,3V, gnd, xxx, io0, rx, tx.

**Comment by Noschvie** [comment]
Have the Olimex device at my desk... installation using [Tasmota Web Installer](https://tasmota.github.io/install/) and auto config was done within a minute, wifi disabled, using ETH and PoE...
Great hardware running great software, Thanks again!

**Reply by barbudor** [reply]
Did you purchased the ISO one or the standard one?

**Reply by Noschvie** [reply]
Olimex ESP32-POE-ISO
